apps/store/src/features/carDealership/MyMoneyConsent/MyMoneyConsent.tsx

```diff
@@ -0,0 +1,27 @@
+import * as Checkbox from '@radix-ui/react-checkbox'
+import { useTranslation } from 'next-i18next'
+import React from 'react'
+import { CheckIcon, Text } from 'ui'
+import { checkboxIndicator, checkboxRoot, consentBody, consentWrapper } from './MyMoneyConsent.css'
+
+const MyMoneyConsent = () => {
```

Comment:
We probably should keep the pattern of using named export for component.

```suggestion
export const MyMoneyConsent = () => {
```